### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by marikrebega**
[Trello Task](https://trello.com/c/vMsONPyQ/75-my-profile-page-redesign)

**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [x] screenshot of feature is added if task has UI

Знімок екрану з 2020-11-06 23-28-33

### CODE REVIEW

**Review by Andrii-Kozak — APPROVED** [review]

**Review by mmarusyk — APPROVED** [review]

File: app/javascript/stylesheets/basic.scss [hunk 1]
```diff
@@ -145,7 +145,7 @@ p {
   min-width: 200px;
   max-width: 600px;
   background-color: rgba(255, 255, 255, 0.8);
-  border-radius: 20px;
+  border-radius: 5px 5px 0 0;
```
Comment (pereslop) [inline app/javascript/stylesheets/basic.scss:148]: Duplicated style

File: app/javascript/stylesheets/user_cabinets.scss [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+      border-radius: 0;
+      border: none;
+      padding: 14px 16px;
+      font-size: 17px;
+      text-align: left;
+      &:first-child {
+        border-radius: 0 5px 0 0;
+      }
+      &:last-child {
+        border-radius: 0 0 5px 0;
```
Comment (pereslop) [inline app/javascript/stylesheets/user_cabinets.scss:23]: Duplicated style

File: app/javascript/packs/user_cabinet.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+$(document).on("turbolinks:load", function () {
+  $(".tablinks").each(function () {
+    $(this).css('background-color', 'none');
+    $(this).removeClass('text-light');
+    if ($(this).attr("href") == window.location.pathname) {
+      $(this).addClass('text-light');
+      $(this).css('background-color', '#448ab5');
```
Comment (pereslop) [inline app/javascript/packs/user_cabinet.js:7]: You cab add this style in sone class in css, and apply with addClass, and the same in line 3

Comment (marikrebega) [reply app/javascript/packs/user_cabinet.js]: done

**Review by chu4 — APPROVED** [review]

**Review by matsevich — APPROVED** [review]

File: app/javascript/stylesheets/basic.scss (hunk 1 above)
Comment (marikrebega) [reply app/javascript/stylesheets/basic.scss:148]: done

**Pull request merged**